content/git/concepts/checkout/checkout.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: 'Checkout'
+Description: 'The git checkout command switches branches or restores files in the working directory to a specific state.'
+Subjects: 
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'GitHub'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+  - 'learn-github-best-practices'
+---
+
+The git **`checkout`** command switches, create and restore branches in the working directory to a specific state. 
+
+It is also possible to checkout to a specific commit without changing branches.
+
+## Syntax
+
+Checkout with branches:
+```pseudo
+git checkout [options] <branch-name>
+```
+Checkout with a specific commit:
+```pseudo
+git checkout <commit-hash>
+```
+## Switch to an existing branch
+
+The following command will switch to an already existing branch, created previously with the [git branch](https://www.codecademy.com/resources/docs/git/branch) command:
+```pseudo
+git checkout existing-branch
+```
+Please, note that from Git 2.23, the new specific `git switch` command has been introduced to switch branches, making it clearer and safer than `git checkout` because it avoids the ambiguity of the latter's multi-purpose nature:
+```pseudo
+git switch existing-branch
+```
```

Comment:
We can skip `the syntax for `git switch` here, as it might get confusing. ``git switch` will eventually have its own entry,` and this can be covered there.

```suggestion
```